### PR TITLE
[JENKINS-35323] Migrate amazon-ecr-plugin to 2.x parent pom

### PR DESCRIPTION
**Pull request by fbelzunc**
@reviewbybees CC @andresrc

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,16 +70,28 @@
...
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
+          <excludeFilterFile>findbugs-exclude.xml</excludeFilterFile>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
   <repositories>
```
Comment (andresrc) [inline pom.xml]: Nit: For future reference, the parent POM provides automatic detection: https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L855-878 (if the file is in the provided path)

**Comment by andresrc** [comment]
:bee:

**Comment by fbelzunc** [comment]
@ndeloof can you review and merge please?

**Comment by fbelzunc** [comment]
I cannot release without this change.

**Pull request merged**